### DISCUSSION TITLE
Journal

### DISCUSSION THREAD

**Discussion by FlorentAdell** [witsa/synapps#774] [Q&A]
Bonjour,

Suite à un problème chez un client,
Il m'a fait remarqué que le journal Redy était bien complet avec pleins de fonctionnalités.
Qui ont toutes ou presque disparu sur la version synapps.
Est-ce que c'est possible d'avoir une version complète de celle ci ?

Notamment les bulles info :
![image](https://github.com/user-attachments/assets/81fccd68-75d6-41fd-8add-1f0b5a42b46d)

La recherche :
![image](https://github.com/user-attachments/assets/9202e98c-a83b-4f5b-abcc-70b0ef52ff59)

Avant :
![image](https://github.com/user-attachments/assets/f78be885-99a6-41c5-88c3-a078391f2cde)

Synapps:

![image](https://github.com/user-attachments/assets/c245fbfd-05d5-480a-96d9-21e095e11071)

**Comment by Raphwit** [comment]
Bonjour, 

Effectivement l'acteur journal de Synapps ne possède pas toutes les fonctionnalités dont dispose le journal intégré dans le REDY, il faut garder à l'esprit que ce sont deux "entités" distinctes.
Ce sont des fonctionnalités qui ne nous avaient pas encore été demandés et elles n'étaient donc pas en priorité dans nos développement futurs.

Il reste pour l'instant d'autres points prioritaires afin d'améliorer l'expérience utilisateur avant que nous nous attaquions aux fonctionnalités manquantes de l'acteur journal mais nous ferons évoluer cet acteur afin d'ajouter les fonctionnalités mentionnés par la suite !


Merci de votre retour !

**Reply by FlorentAdell** [reply]
Salut Raph, ok merci de ta réponse, c'est vrai que je garde ça en tête, c'est dommage qu'il soit trop synthétique malgré qu'il soit plus jolis
